deploy/values.yaml

```diff
@@ -0,0 +1,173 @@
+  - url: 'mc://node5.test.mobilecoin.com:443/'
+    txSource: 'https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node5.test.mobilecoin.com/'
+  - url: 'mc://node6.test.mobilecoin.com:443/'
+    txSource: 'https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node6.test.mobilecoin.com/'
+  - url: 'mc://node7.test.mobilecoin.com:443/'
+    txSource: 'https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node7.test.mobilecoin.com/'
+  - url: 'mc://node8.test.mobilecoin.com:443/'
+    txSource: 'https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node8.test.mobilecoin.com/'
+
+  - url: 'mc://node1.consensus.mob.staging.namda.net:443/'
+    txSource: 'https://s3-eu-central-1.amazonaws.com/staging-namda-payments-ledger/node1.consensus.mob.staging.namda.net/'
+  - url: 'mc://node2.consensus.mob.staging.namda.net:443/'
+    txSource: 'https://s3-eu-central-1.amazonaws.com/staging-namda-payments-ledger/node2.consensus.mob.staging.namda.net/'
+
+ledgerScanner:
+  image:
+    repository: mobilecoin/reserve-auditor
+    pullPolicy: Always
+    tag: v0.0.0-test
+  resources:
+    requests:
+      cpu: 100m
+      memory: 65Mi
+
+httpServer:
+  image:
+    repository: mobilecoin/reserve-auditor
+    pullPolicy: Always
+    tag: v0.0.0-test
+  resources:
+    requests:
+      cpu: 100m
+      memory: 65Mi
+
+persistence:
+  mobilecoind:
+    enabled: true
+    pvcSpec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 128Gi
+      storageClassName: fast
+      volumeMode: Filesystem
+
+  ledger:
+    enabled: true
+    pvcSpec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 128Gi
+      storageClassName: fast
+      volumeMode: Filesystem
+
+  auditor:
+    enabled: true
+    pvcSpec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 128Gi
+      storageClassName: fast
+      volumeMode: Filesystem
+
+gnosisConfig:
+  safes:
+  - safe_addr: "0x36c4A3e3838974bb94Da7fC836987A68C7c0D34D"
+    api_url: https://safe-transaction.goerli.gnosis.io/
+    token_decimals_max: 6
+    tokens:
+      - token_id: 1
+        token_type: ERC20_TRANSFER
+        eth_token_contract_addr: "0xeC76FbFD75481839e456C4cb2cd23cda813f19B1"
+        name: gRSV
+        symbol: Token
+        decimals: 18
+        logo_uri: ""
+        aux_burn_contract_addr: "0x89d2c2C7853AD6015d50f39A615196e17f13Cf59"
+        aux_burn_function_sig: [199, 111, 6, 53]
+
+imagePullSecrets:
+  - name: docker-credentials
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
```

Comment:
Calling out this and other commented-out values to confirm they're here intentionally

Reply:
Yes, those are added by the default scaffolding as examples